CMakeLists.txt

```
@@ -1129,6 +1129,7 @@ ENDIF()
 # version has the necessary patches to make it work.
 SET(SYSTEM_LIBRARIES
   CURL
+  EDITLINE
```

Percona Server release packages used to be built with `-DWITH_READLINE=ON`. @EvgeniyPatlan , please confirm.
So, I think this should be the default library when we specify either none of the `-DWITH_READLINE=XXX / -DWITH_EDITLINE=XXX / -DWITH_SYSTEM_LIBS=XXX` or `-DWITH_SYSTEM_LIBS=ON`
```suggestion
  READLINE
```
I might have mislead you before when I was still testing this, but this looks like the proper solution.
Also, it doesn't look like this change belongs to this commit.

---

I've already removed this change, but Satya's latest bugfix accidentally readded it. Removing.